Sources/MessagingPush/Config/MessagingPushConfigOptions.swift

```diff
@@ -0,0 +1,20 @@
+import CioInternalCommon
+
+/// Defines configuration options for the Customer.io Push Messaging module.
+///
+/// Use `MessagingPushConfigBuilder` for constructing its instances. For detailed usage, see builder class documentation.
+public struct MessagingPushConfigOptions {
+    let cdpApiKey: String
```

Comment:
Why not public?

Reply:
Previously we didn’t expose it for customers so they don’t get confused when setting config. But with builder pattern, I see that’s not required, so updated to `public`.